solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/pom.xml

```diff
@@ -102,6 +102,10 @@
             <artifactId>log4j-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
```

Comment:
Why was this added?

Reply:
We are using org.apache.commons.lang3.exception.ExceptionUtils in binder. 

Earlier this dependency was transitively available. When we removed dependency on java-cfenv we need to explicitly add this dependency.